### ISSUE TITLE
a11y: support reader mode across all Zooniverse pages

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#7256]
## Package
- app-project
- app-root
- lib-classifier
- lib-content

## Describe the bug
Reader mode is supported by some Zooniverse web pages but not all. Notably, the home page and Projects page don't support reader mode.

## Screenshots
These videos show reader mode, with Read Aloud enabled, in Firefox for the About page and a random project classify page.

https://github.com/user-attachments/assets/b5edb229-da59-47f5-8904-3ad228b2c547

https://github.com/user-attachments/assets/97f6af09-b5df-4ebd-a9fc-30404e9d76de

## Expected behavior
Read Aloud should work for the home page, project pages, volunteer profile/stats pages, the projects listing page and informational pages.

## Additional context
>These features may not appear for pages that are not well-formed (lack a main region, are nothing but div-soup, etc.). If they don’t appear for your content, that’s a signal that you may have work to do on the underlying HTML. Any fixes you make would likely result in a better experience for everyone. 

[Your browser can already speak a page](https://adrianroselli.com/2026/03/your-browser-can-already-speak-a-page.html) by Adrian Roselli.
